package.json

```diff
@@ -2,37 +2,44 @@
   "name": "stack",
   "version": "0.1.0",
   "private": true,
-  "main": "public/electron.js",
+  "main": "src/electron.js",
   "dependencies": {
     "electron-is-dev": "^1.0.1",
+    "preact": "^8.3.1",
     "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "react-scripts": "1.1.5",
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
-    "react-dev": "BROWSER=none npm start",
-    "electron-dev": "nodemon --watch public/electron.js node_modules/.bin/electron .",
-    "all-dev": "concurrently \"BROWSER=none npm start\" \"wait-on http://localhost:3000 && electron .\"",
-    "electron-pack": "build build/electron.js",
+    "start": "npm run all-dev",
+    "build": "parcel build ./src/index.html ./src/screenshot.html --out-dir=build --public-url ./",
+    "all-dev": "concurrently \"BROWSER=none npm run preact-dev\" \"wait-on http://localhost:3000 && electron .\"",
```

Comment:
On my machine, running `npm run all-dev` only works the first time. Subsequent runs do not start the electron application properly.
Running `concurrently \"electron . && wait-on http://localhost:3000\" \"BROWSER=none npm run preact-dev\"` solves this behaviour and starts the application properly.

Reply:
didn't use all-dev for a while can confirm it's not working anymore. problem is, that we now have two entry points and `http://localhost:3000` is not correct, but `http://localhost:3000/index.html` is. Fixed that, please check.

Reply:
👍 All good now!